packages/fhir-care-team/src/components/CreateEditCareTeam/tests/utils.test.ts

```diff
@@ -1,174 +0,0 @@
-import { getPatientName, submitForm } from '../utils';
```

Comment:
is this intentional?

Reply:
yes, I think the tests in this fiil did not perform meaningful assertions. Furthermore I think what it was attempting to test is now better handled by functional tests in `form.unmocked.test.ts`